Cargo.toml

```diff
@@ -262,6 +262,7 @@ members = [
   "sdk/crates/sdk",
   "types",
   "types/abi",
+  "client/packages/executor-gui/src-tauri"
```

Comment:
@3h4x if I remove this line, I won't be able to build the tauri project.

Reply:
this is no good then... 
how will CI know that this change is not our blockchain change?
with any change to tauri CI will want to do runtime upgrade. we need to solve it before merging.
I guess we need to move it to seperate repo then